### DISCUSSION TITLE
Renaming rooms

### DISCUSSION THREAD

**Discussion by Adrian-ARKCLS** [Zuehlke/poinz#357] [Q&A]
We started to use poinz.app for internal planning poker which we are finding very useful and simple to use.  Thanks for providing this tool.  One question.  Is there a way to rename a room?

**Comment by xeronimus** [comment]
Hello Adrian,

sorry for the late reply - it's been a busy week.

Currently the "Name" is a unique id (that also is the direct url to the room) and never changes during the lifetime of a room.

If I understand the use case a bit better, I might be able to provide you with a workaround:

first thoughts:
- if this is more of a "one time thing", it might be an idea to create a new room and transfer all the stories to the new room.  I guess, you'd want also to copy the already estimated stories and their estimation value / votes over to the new room? at least all this would be achievable through a script that sends the correct commands to the backend via websocket.
- if this is a feature that you would want to use on a regular basis, I'd need to check what the implications would be of changing the unique Id of a room.
- the most promising solution would be a "import stories" feature. Poinz already provides the export into a json file (currently it has all the needed information in it except deleted stories - but this could be easily implemented). Thus the workflow would be: export the stories from your current room, create a new room with the desired name, import the stories

**Reply by xeronimus** [reply]
I started Implementing the "import from poinz export file", you can check it out here: #358

**Reply by Adrian-ARKCLS** [reply]
Many thanks for your response.  Our team is new to poinz.app and we didn't initially named the room sensibly such that we couldn't remember the room name when we wish to return a couple of week later.  We eventually did remember and wanted to rename it such a that we could easily remember in the future.  So in this case it is just a "one time thing" because we won't make that mistake again.  The workaround you suggested would be fine for us.  Our team is new to git and github having worked for many years using mercurial and hosting our repositories locally.   We really like your easy to use planning poker tool and hope to use it regularly as we start to get up to speed with software development through github.

**Reply by Adrian-ARKCLS** [reply]
I see your are a pilot also.

**Reply by xeronimus** [reply]
Hi Adrian,

if you send me an [email ](mailto:xeronimus@gmail.com) with your current room-id and the desired new room-id (and ideally a time window where you don't need the room), then I can rename your room manually.

However, your message triggered a nice addition to Poinz, in the next release, we will be able to import stories from json files  :-D
